spec/controllers/posts_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe PostsController, type: :controller do
+
+  FactoryGirl.define do
```

Comment:
This should be in spec/factories/posts.rb as per: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#defining-factories